docs/02_cleos/03_command-reference/wallet/import.md

```diff
@@ -1,9 +1,55 @@
+## Command
+cleos wallet import [OPTIONS]
+
+**Where**
+* [OPTIONS] = See Options in Command Usage section below. 
+
+**Note**: The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Imports private key into wallet
+Imports private key into wallet. This command will launch `keosd` if it is not already running. 
 
-## Positionals
-None
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos wallet import` command:
 
-## Options
+### Positionals:
+- none
+### Options
+- `-h,--help` - Print this help message and exit
 - `-n, --name` _TEXT_ - The name of the wallet to import key into.
-- `--private-key` _TEXT_ - Private key in WIF format to import.
\ No newline at end of file
+- `--private-key` _TEXT_ - Private key in WIF format to import.
+
+## Requirements
+* Install the currently supported version of `cleos` and `keosd`.
+[[info | Note]]
+| The `cleos` tool and `keosd` wallet is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos comand line tool and the keosd key store or wallet. 
```

Comment:
is bundled -> are bundled

`keosd` is not a wallet. Consider simplifying to:
The \`cleos\` tool and \`keosd\` wallet is bundled -> \`cleos\` and \`keosd\` are bundled

The phrase "or wallet" is not accurate. Consider simplifying to:
... will also install the \`cleos\` and \`keosd\` command-line tools.